aws/diff_suppress_funcs.go

```diff
@@ -115,14 +115,6 @@ func suppressCloudFormationTemplateBodyDiffs(k, old, new string, d *schema.Resou
 	return normalizedOld == normalizedNew
 }
 
-func suppressRoute53ZoneNameWithTrailingDot(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Ah yes I wasn't sure what the best course of action was here since there are changes spread out across different resources and changes in the PR will break tests that are addressed in the other PRs. If this PR should take precedence over the other ones,  do it make more sense to bring the method back here to get the build to pass and then later remove it when it's no longer referenced?

Reply:
I'm totally okay reviewing the other resource changes as part of this PR, since everything is all inter-connected. 👍

Reply:
sounds good, thanks!